src/project-tests/pomodoro-clock-tests.js

```diff
@@ -78,476 +84,718 @@ export default function createPomodoroClockTests() {
-          setTimeout(_ => {
-            const secondsAfter = getSeconds(document.getElementById("time-left").innerHTML);
+          setTimeout(() => {
+            const secondsAfter = getSeconds(
+              document.getElementById('time-left').innerHTML
+            );
             if (secondsAfter === secondsBefore) {
-              reject(new Error("Pomodoro has started but time displayed is not changing"));
+              reject(
+                new Error('Pomodoro has started but time displayed is not ' +
+                'changing')
+              );
               return;
             }
             // Pause the pomodoro
-            clickButtonsById([_start_stop]);
+            clickButtonsById([_startStop]);
             // wait another 1.5 seconds to be sure value has not changed
-            setTimeout(_ => {
-              const secondsAfterPause = getSeconds(document.getElementById("time-left").innerHTML);
-              if (secondsAfter === secondsAfterPause)
+            setTimeout(() => {
+              const secondsAfterPause = getSeconds(
+                document.getElementById('time-left').innerHTML
+              );
+              if (secondsAfter === secondsAfterPause) {
                 resolve();
-              else
-                reject(new Error("Pomodoro has paused but time continued elapsing"));
+              } else {
+                reject(
+                  new Error('Pomodoro has paused but time continued elapsing')
+                );
               }
-            , 1500);
+            }, 1500);
           }, 1500);
         });
       });
 
-      it('21. If the timer is paused and I click the element with id="start_stop", the countdown should resume running from the point at which it was paused.', function() {
+      reqNum++;
+      it(`${reqNum}. If the timer is paused and I click the element with 
+      id="start_stop", the countdown should resume running from the point at 
+      which it was paused.`, function() {
         this.timeout(5000);
         // start the pomodoro
-        clickButtonsById([_start_stop]);
-        const secondsBefore = getSeconds(document.getElementById("time-left").innerHTML);
+        clickButtonsById([_startStop]);
+        const secondsBefore = getSeconds(
+          document.getElementById('time-left').innerHTML
+        );
         return new Promise((resolve, reject) => {
           // wait 1.5 seconds then see if displayed time has changed
-          setTimeout(_ => {
-            const secondsAfter = getSeconds(document.getElementById("time-left").innerHTML);
+          setTimeout(() => {
+            const secondsAfter = getSeconds(
+              document.getElementById('time-left').innerHTML
+            );
             if (secondsAfter === secondsBefore) {
-              reject(new Error("Pomodoro has started but time displayed is not changing"));
+              reject(
+                new Error('Pomodoro has started but time displayed is not ' +
```

Comment:
Last comment on this ;-) I'll let you search for any other places you need to put the parameter on its own line.